### PR TITLE
Feat createTrip

### PR DESCRIPTION
**Pull request by bertacume**

### CODE REVIEW

File: thesis-fe/src/components/apollo/mutations/create_trip.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import gql from "graphql-tag";
+
+const CREATE_TRIP = gql`mutation createTrip($trip: TripInput!) {
+  createTrip(trip: $trip, userID: "5bffbe8ff59a127f1afadf18") {
```
Comment (ArtyBMPS) [inline thesis-fe/src/components/apollo/mutations/create_trip.js]: userID should not be hardcoded

**Review by ArtyBMPS — APPROVED** [review]

**Review by ArtyBMPS — CHANGES_REQUESTED** [review]
please fix the add trip id

File: thesis-fe/src/components/apollo/mutations/create_trip.js (hunk 1 above)
Comment (bertacume) [reply thesis-fe/src/components/apollo/mutations/create_trip.js]: True, I'm gonna delete this field and as its not required the mutation should work

File: thesis-fe/src/components/apollo/mutations/create_trip.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import gql from "graphql-tag";
 
 const CREATE_TRIP = gql`mutation createTrip($trip: TripInput!) {
-  createTrip(trip: $trip, userID: "5bffbe8ff59a127f1afadf18") {
+  createTrip(trip: $trip) {
```
Comment (ArtyBMPS) [inline thesis-fe/src/components/apollo/mutations/create_trip.js:4]: Great

**Review by ArtyBMPS — APPROVED** [review]

**Pull request merged**